bundles/org.openhab.binding.rotel/src/main/resources/OH-INF/thing/a14.xml

```diff
@@ -18,12 +18,15 @@
 			<channel id="treble" typeId="treble"/>
 			<channel id="frequency" typeId="frequency"/>
 			<channel id="brightness" typeId="brightness"/>
+			<channel id="tcbypass" typeId="tcbypass"/>
+			<channel id="balance" typeId="balance"/>
+			<channel id="speakera" typeId="speakera"/>
+			<channel id="speakerb" typeId="speakerb"/>
```

Comment:
Please add these 4 channels on models A11 and A12 (and update model definition).
I will consider other models in a next PR.

Reply:
I do so